ui/app/src/components/App.tsx

```diff
@@ -20,10 +20,23 @@ import CrafterCMSNextBridge from './CrafterCMSNextBridge';
 
 const AsyncVideoPlayer = React.lazy(() => import('./AsyncVideoPlayer'));
 
+const playerOptions = {
```

Comment:
Get rid of everything AsyncVideoPlayer from this file. Leave it with just the CrafterCMSNextBridge and a hello world message inside.